packages/react-bindings/src/contexts/StreamVideoContext.tsx

```diff
@@ -39,12 +39,6 @@ export const StreamVideo = ({
   const prevClient = useRef<StreamVideoClient | undefined>(undefined);
 
   useEffect(() => {
```

Comment:
I made a mistake with the connect-disconnect, I only noticed when testing the tutorial